docs/framework/How-to-Get-Updates/How-to-get-updates

```diff
@@ -0,0 +1,18 @@
+## How to Get Security and Quality Rollup .NET Framework updates:
```

Comment:
Why don't these files have file extensions? They should be `.md` files. Otherwise, they won't render properly, nor will we get good markdown linting.